pootle/static/js/shared/components/KeyboardHelpDialog.js

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (C) Zing contributors.
+ *
+ * This file is a part of the Zing project. It is distributed under the GPL3
+ * or later license. See the LICENSE file for a copy of the license and the
+ * AUTHORS file for copyright and authorship information.
+ */
+
+import React from 'react';
+import { showModal } from './Modal';
+
+import './KeyboardHelpDialog.css';
+
+let keyboardHelpOpen = false;
+
+export function showKeyboardHelpDialog() {
+  if (keyboardHelpOpen) {
+    return;
+  }
+
+  keyboardHelpOpen = true;
+  showModal({
+    title:
+      <span>
+        Keyboard Help (press <kbd>F1</kbd> to open;
```

Comment:
As you see _Close dialog_ is a generic one (like in Zing), and as such it is listed under the global section. This is what other apps such as Trello do too. Therefore I would move that down to the general section.

![pantaila-argazkia 2016-10-26 10 50 25](https://cloud.githubusercontent.com/assets/16768/19719383/f29aed2e-9b69-11e6-85e8-52b5fe956755.png)